common/ipfs_util.py

```diff
@@ -8,7 +8,7 @@
 
 import logging
 
-from common.constant import IPFS_URL
```

Comment:
@pratik-vii @prashantramangupta  why this has changes from constant to parse_event